masakari-controller/controller/masakari_controller.py

```diff
@@ -47,10 +46,14 @@
 import controller.masakari_starter as starter
 from controller.masakari_util import RecoveryControllerUtil as util
 from controller.masakari_util import RecoveryControllerUtilDb as util_db
+from controller.masakari_util import LogProcessBeginAndEnd
+from oslo_log import log as oslo_logging
 import controller.masakari_config as config
 import controller.masakari_worker as worker
 import db.api as dbapi
 
+log_process_begin_and_end = LogProcessBeginAndEnd()
+LOG = oslo_logging.getLogger('controller.masakari_controller')
 
```

Comment:
I understand.  In that case, writing the module path directly make sense. I have another idea to enable the file to use \__name__ but it will change the init script. I think the change should be in different patch.

I don't think setting 'controller.masakari_controller' to \__name__ is good since when we want to use \__name__ in other place \__name__ doesn't work well.

Another idea is creating a new file that only calls masakari() in the file and init script calls the file. The reason \__name__ is set to "\__main__" here is the init script for service command calls this file as a command argument directly.